guides/v3.5.0/models/index.md

```diff
@@ -54,9 +54,25 @@ more information about how components get model data, see the
 [Specifying a Route's Model](../routing/specifying-a-routes-model/)
 guide.
 
+At first, using Ember Data may feel different than the way you're used
+to writing JavaScript applications. Many developers are familiar with
+using AJAX to fetch raw JSON data from an endpoint, which may appear
+easy at first. Over time, however, complexity leaks out into your
+application code, making it hard to maintain.
+
+With Ember Data, managing models as your application grows becomes both
+simple _and_ easy.
```

Comment:
```suggestion
simpler _and_ easier.
```
It can still be hard to use Ember Data, sometimes. I think maybe the -er is a bit more realistic and kinder to new learners.